discord/tool.gpt

```diff
@@ -106,8 +106,8 @@ Share Tools: List Channels, Search Channels, Get Channel History, Get Thread His
 Share Context: Discord Context
 Param: channelid: the ID of the channel containing the thread
 Param: guildid: the ID of the guild that the channel belongs to
-Param: threadid: the ID of the thread to send the message to
 Param: text: the text content to send
+Param: messageid: the ID of the message to create a thread on
```

Comment:
Seems like this should be marked as "optional", since it doesn't look like it's necessary to send a new message on a thread.

Reply:
yeah, I am afraid that optional might break it for new message. We should always have message id in context when replying in the thread. although it is not used inside thread.